site/docs/master/customize-installation.md

```diff
@@ -12,6 +12,14 @@
   - [Generate YAML only](#generate-yaml-only)
   - [Additional options](#additional-options)
   - [Optional Velero CLI configurations](#optional-velero-cli-configurations)
+    - [Enabling shell autocompletion](#enabling-shell-autocompletion)
+    - [Bash on Linux](#bash-on-linux)
+      - [Install bash-completion](#install-bash-completion)
+      - [Enable Velero CLI autocompletion for Bash on Linux](#enable-velero-cli-autocompletion-for-bash-on-linux)
+    - [Bash on macOS](#bash-on-macos)
+      - [Install bash-completion](#install-bash-completion-1)
+      - [Enable Velero CLI autocompletion for Bash on macOS](#enable-velero-cli-autocompletion-for-bash-on-macos)
+      - [Autocompletion on Zsh](#autocompletion-on-zsh)
```

Comment:
"Autocompletion on Zsh" should be at the same level as the two Bash ones, not nested under "Bash on macOS" (same for section header there) :)

Reply:
didn't follow the
> same for section header there

part. Can you please clarify?

Reply:
meaning remove a "#" from the "Autocompletion on Zsh" section header

Reply:
done. markdown is hard :-|

Reply:
yep, this stuff is tedious :/